interop_binaries/setup.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+timeout 5m bash -c 'until pg_isready -U postgres; do sleep 1; done'
```

Comment:
Consider running this script with `set -e` or perhaps better adding `|| "timeout waiting for postgres to be ready" && exit 1`. Otherwise the script will fall through on failure.

This is a nit since we'll be able to see the results in the logs anyways, can take it or leave it.

Reply:
I was able to resolve the issue with this change !
I have tested it in my local environment for both release/0.5 and 0.6.0-prerelease-1, and both passed successfully.

Thank you .